_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+They are setting up the Product Backlog and decide the business value of each ticket and with that the ticket order.
+That means, the PO is responsible for the Return of Investment (ROI).
+They decide if fixing a bug is more important than developing a new feature.
+They spend a lot of time maintaining the Product Backlog, which means creating new tickets and prioritising them as well as keeping the backlog up to date.
+Several techniques exist of how to prioritise a Backlog.
+The technique to be used mainly depends on the product, the team is working on.
+
+To create the tickets the PO is also responsible for gathering all the requirements as well as to talk to the stakeholders.
+Managing all the stakeholders for a product can be quite time consuming.
+But there are some techniques that can help the PO to keep an overview.
+
+The PO's superior task is to create a common product vision and to steer the product release.
+They are also responsible for accepting (or not ;) ) the developed tickets from the Development Team.
+As you can see, the Product Owner is a very important and complex role in Scrum and so it’s no wonder that several trainings exist one should participate in before agreeing on this job.
+
+# Scrum Master
+
+{% image blog-scrum2-scrummaster.jpg 30% left %}{% endimage %}
+
+The Scrum Master is normally the most unclear role for people who are new to Scrum.
+The concept is just not existing in other frameworks and therefore the value of this role is often underestimated.
+After all, the Scrum Master is not directly adding development power to the team, so why should a company still invest in an extra employee?
+
+The superior task of a Scrum Master is to protect the Scrum Team!
+And yes, this includes developers and PO.
+One example that most software developers probably understand is the problem of scope creep.
+The developers are working on their feature, when some manager steps into the room and forces someone to take care of an ultra-urgent topic.
+The developer is distracted from his actual task and since it was a manager who told them to go work on something else, they usually do it.
+Not in Scrum!
+In Scrum it would be the task of the Scrum Master to step inbetween the manager and the developer.
+The Scrum Master would tell the manager politely to go and talk to the PO, who will create a ticket for his request and sort it into the Backlog according to its business value.
+If it really is ultra-urgent, it will be solved within the next sprint.
+Only in real blocker situations (like a live-platform being down) a running sprint would be interrupted.
+And ONLY the PO has the right to decide this.
+
+That little story already leads to the third biggest part of the Scrum Masters time: Coaching.
+He is responsible to coach not only the team and the PO on Scrum and agile development topics, but also all other departments in the company that have to interact with the Scrum Teams.
```

Comment:
His responsibility is to coach not only the team and the PO on Scrum and agile development topics, but also all other departments in the company that have to interact with the Scrum Teams.